tg/views/__init__.py

```diff
@@ -307,15 +311,15 @@ def draw(
 
         self._refresh()
 
-    def _msg_attributes(self, is_selected: bool) -> Tuple[int]:
+    def _msg_attributes(self, is_selected: bool) -> Tuple[int, ...]:
```

Comment:
what ... means in type?

Reply:
this annotation for tuple of variable size. plz see the [docs](https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html#built-in-types) mypy can't detect that tuple contains exactly 3 elements, that's why I am using it. 
Anyway, it is much better that wrong annotation `Tuple[int]:` which means tuple with one element inside